test/integration/ae_mdw_web/controllers/aex9_controller_test.exs

```diff
@@ -317,6 +452,22 @@ defmodule Integration.AeMdwWeb.Aex9ControllerTest do
         assert token_symbol == symbol
       end)
     end
+
+    @tag timeout: 180_000
+    # @tag :iteration
+    @tag :skip
+    test "gets balances for each account with aex9 presence", %{conn: conn} do
+      Model.Aex9AccountPresence
+      |> Database.all_keys()
+      |> Enum.map(fn {account_pk, _txi, _contract_pk} ->
+        :aeser_api_encoder.encode(:account_pubkey, account_pk)
+      end)
+      |> Enum.uniq()
+      |> Enum.each(fn account_id ->
+        conn = get(conn, "/aex9/balances/account/#{account_id}")
+        assert balances_response = json_response(conn, 200)
```

Comment:
Can we pattern-match the response structure here to avoid an unused variable? Perhaps something like `%{} = ...` if we don't want to map specific keys

Reply:
done